api4/api.go

```diff
@@ -77,6 +77,8 @@ type Routes struct {
 
 	LDAP *mux.Router // 'api/v4/ldap'
 
+	Elasticsearch *mux.Router // 'api/v4/elasticsearch'
```

Comment:
I know it's correct, but the capitalization of "Elasticsearch" will always bug me